### DISCUSSION TITLE
Migrating off Isar

### DISCUSSION THREAD

**Discussion by pstreck** [isar-community/isar#111] [General]
The future of Isar does not look promising, as disappointing as this is I am looking to make the move to another database.

Has anyone started or completed the process of migrating off Isar to another solution?
What did you migrate to or plan to?
What challenges or problems did you face in migrating an existing app database?

**Comment by jbarasa** [comment]
I tried object box but had several issues with the relationships so i decided to fork the Isar project and update and fix a few packages

**Reply by UnluckyY1** [reply]
We migrated from ObjectBox to Isar two years ago due to performance issues, and based on my experience, I strongly discourage using ObjectBox. Isar v3 has been very stable (at least for our use case). I currently use a mix of the isar-community version and a custom fork I created to auto-save links. So far, I’m really satisfied with the results. You can check my fork here: https://github.com/UnluckyY1/isar/tree/3.1.0-fixed
